bin/existing-eks-awsnative-observability.ts

```diff
@@ -1,4 +1,4 @@
-import ExistingEksAwsNativeObservabilityConstruct from '../lib/existing-eks-awsnative-observability-construct';
+import ExistingEksAwsNativeObservabilityConstruct from '../lib/existing-eks-awsnative-observability';
```

Comment:
Why is `ExistingEksAwsNativeObservabilityConstruct` still having `Construct` ? We want to totally get rid of word Construct with `Pattern`. Also we dont want `-construct` to be removed. We would rather want it to be renamed to `-pattern`.